.gitignore

```diff
@@ -12,3 +11,4 @@ docs/
 npm-debug.log
 /.idea/
 .env
+dist/*
```

Comment:
I thought we had this before, we don't want to commit dist/ every commit as it would be noisy. That's all this line is about.